assets/operator/okd-x86_64/jenkins/imagestreams/jenkins-centos.json

```diff
@@ -63,7 +63,7 @@
 				},
 				"from": {
 					"kind": "DockerImage",
-					"name": "quay.io/openshift/origin-jenkins:latest"
+					"name": "quay.io/openshift/origin-jenkins@sha256:1c74f4727908c9455aafb0b0ebfecfd9dddc30b3383a81c3eb834aeddc784cbd"
```

Comment:
curious this does not line up with https://github.com/openshift/jenkins/blob/master/openshift/imagestreams/jenkins-centos.json#L58

will revert in a bit